src/_includes/tools/debug-prod-js-code.md

```diff
@@ -54,12 +54,12 @@ To debug in Firefox:
    
 3. Reload the app. The **Debugger** tab shows source-mapped files.
 
-### Safari {#dart2js-debugging-safari}
+### Safari {:#dart2js-debugging-safari}
```

Comment:
What's the respon for switching this to : as opposed to the standard #?

Reply:
Patrice asked the same question :D. For one was consistency with the style we used when adding classes and IDs to elements, but I think I had another reason at the time. I just forgot what that reason was... It was maybe to avoid potential conflicts with another syntax or something?

I'll try to remember and if I can, document it, otherwise consider going back to the shorter syntax consistently.